### PR TITLE
chore: add OOV abbreviation

### PR DESCRIPTION
**Pull request by SponsoredByPuma**

### CODE REVIEW

**Comment by pkeilbach** [comment]
Closes #179

**Comment by SponsoredByPuma** [comment]
@pkeilbach I cannot assign you as a reviewer, but this could be ready to merge

File: docs/lectures/language_models.md [hunk 1]
```diff
@@ -864,7 +864,7 @@ Since the perplexity is the inverse probability of the set set, the lower the pe
 
 ## Out of Vocabulary Words
```
Comment (pkeilbach) [inline docs/lectures/language_models.md:865]:
```suggestion
## Out of Vocabulary (OOV) Words
```

**Review by pkeilbach — CHANGES_REQUESTED** [review]

**Review by pkeilbach — APPROVED** [review]
LGTM! 🏅

**Pull request merged**